tests/unit/test_network_connections.py

```diff
@@ -3570,6 +3570,32 @@ def test_ipv6_dns_without_ipv6_config(self):
             0,
         )
 
+    def test_ipv6_dns_when_ipv6_method_is_ignore(self):
+        """
+        Test that configuring IPv6 DNS is not allowed when IPv6 method is ignore.
```

Comment:
What is the difference in the test scenario compared to the previous test? "test_ipv6_dns_without_ipv6_config". Please do not use "IPv6 method is ignore" in this context, I believe I explained this in another comment (you might not have read it, yet).

Reply:
Ah, I see. So the possible scenarios are the combination of ipv6_disabled:true|false x auto6:true|false x static_ipv6_address:true|false

So is this table correct? The last column is the result of the previous columns:

<table>
<tr>
	<td>ipv6_disabled
	<td>auto6
	<td>static ipv6 address
	<td>IPv6 nameserver support
<tr>
	<td>true
	<td>*
	<td>*
	<td>no
<tr>
	<td>false
	<td>false
	<td>false
	<td>no
<tr>
	<td>false
	<td>true
	<td>*
	<td>yes
<tr>
	<td>false
	<td>*
	<td>true
	<td>yes
</table>

Reply:
Thanks for making it so clear. Yes, the table is correct.

Reply:
ok, so you called row 2 "disabled" so it might make sense to call the previous test  "test_ipv6_dns_with_ipv6_disabled" and this one "test_ipv6_dns_without_ipv6_configuration" (just a suggestion). 

Btw. I did not look into this but it would also be good to add test cases for the other scenarios from the table that are not specific to DNS, so for example ipv6_disabled:true and auto6:true should probably also raise a validation error (not sure if they do or don't). But this is not related to the DNS feature.

Reply:
FYI,  `ipv6_disabled:true` and `auto6:true` are mutually exclusive, this has already been implemented, https://github.com/linux-system-roles/network/blob/main/module_utils/network_lsr/argument_validator.py#L655

Reply:
I created a separate PR to validate the other scenarios from the table,
https://github.com/linux-system-roles/network/pull/425